### DISCUSSION TITLE
Reachable workspace

### DISCUSSION THREAD

**Discussion by Betancourt20** [stack-of-tasks/pinocchio#1886] [Q&A] [answered]
Hello,

I wonder if there is a method for obtaining the reachable workspace of a robot manipulator? or any hint for obtaining it?
Thanks in advance

**Comment by jcarpent** [comment]
Good question @Betancourt20.
There is no direct way of computing it, but it should be easy to implement such an algorithm by taking inspiration from https://arxiv.org/pdf/2211.17054.pdf. @vpadois Any advice to provide on this topic?

**Comment by vpadois** [comment, marked as answer]
Hello @Betancourt20 @jcarpent 
[pycapacity](https://pypi.org/project/pycapacity/) is a good option for that. It was used for the paper mentionned by @jcarpent There is some work in progress by @askuric on this topic which may be incorporated in pycapacity in the coming months.

**Comment by Betancourt20** [comment]
Hello @jcarpent @vpadois 

Hope you're doing great! 

Thank you so much for your advices. At the end, I found an "easy" way to do it. For sure it is not the optimal but for my purposes it is enough.

Best regards,

Julio
![output](https://user-images.githubusercontent.com/23742018/226349614-f80c3660-124b-4b80-844c-3ea2c0953d5f.png)

**Reply by jcarpent** [reply]
Thanks for your feedback. It is very helpful for the other Pinocchio users
